slack_notification/README.md

```diff
@@ -4,26 +4,50 @@ Hook into platform workflows and post notifications to Slack.
+
+```bash
+curl -X POST -H "Authorization: Bearer xoxb-YOUR-TOKEN" \
+     -H "Content-Type: application/json" \
+     -d '{
+           "channel": "#channel-name",
+           "text": "Hello from Deploybot!"
+         }' \
+     https://slack.com/api/chat.postMessage
 ```
 
-3. Add the example `slack_notification.php` script to the `private` directory in the root of your site's codebase, that is under version control. Note this is a different `private` directory than where the `secrets.json` is stored.
-4. Add Quicksilver operations to your `pantheon.yml`
-5. Test a deployment and see the notification in the Slack channel associated with the webhook.
+### Add the OAuth token to Pantheon Secrets
+
+1. Install the [Terminus Secrets Manager Plugin](https://docs.pantheon.io/guides/secrets#installation).
+1. Set the secret with the following command: `terminus secret:site:set <site> <secret-name> <oauth-token> --scope=web`
+  - Replace `<site>` with your site name (e.g. `my-site`).
+  - Replace `<secret-name>` with the name of your secret that you will use in the code. In the example script, this is set to `slack_deploybot_token`.
+  - Replace `<oauth-token>` with the Bot User OAuth Token copied from the above steps.
+1. Add the example `slack_notification.php` script to the `private` directory in the root of your site's codebase, that is under version control.
+1. Update the script to set the `$slack_channel` (on line 8) to whatever channel you wish to push notifications to.
+1. Update the secret name in the `pantheon_get_secret` function call on line 151 (if you are using a secret that is not named `slack_deploybot_token`).
+1. Make any other customizations of the script as you see fit.
+1. Add Quicksilver operations to your `pantheon.yml` (see the [example](#example-pantheonyml) below).
+1. Test a deployment and see the notification in the Slack channel associated with the webhook.
+
+Optionally, you may want to use the `terminus workflows watch` command to get immediate debugging feedback or use the [Workflow Logs](https://docs.pantheon.io/workflow-logs) to return any debugging output. 
+
+**Note:** The example `slack_notification.php` script uses [message attachments](https://api.slack.com/reference/messaging/attachments) to keep the colored sidebar while using the updated API. This can be swapped out in favor of a [block-based](https://api.slack.com/reference/block-kit/blocks) approach entirely if that cosmetic detail is not important to you (we use blocks for everything else).
 
-Optionally, you may want to use the `terminus workflows watch` command to get immediate debugging feedback. You may also want to customize your notifications further. The [Slack API](https://api.slack.com/incoming-webhooks) documentation has more on your options.
+For an example of the script that doesn't use attachments at all, see [@JBCSU's PR](https://github.com/pantheon-systems/quicksilver-examples/pull/176/files).
```

Comment:
Linking to an unmerged PR seems dubious.

Reply:
I think it'd be better to show both in the php file. Maybe with one commented out.